pallets/did/src/lib.rs

```diff
@@ -81,9 +91,19 @@ pub mod pallet {
 			+ Dispatchable<Origin = <Self as Config>::Origin, PostInfo = PostDispatchInfo>
 			+ GetDispatchInfo
 			+ DeriveDidCallAuthorizationVerificationKeyRelationship;
-		type DidIdentifier: Parameter;
+		type DidIdentifier: Parameter + Default;
+		#[cfg(not(feature = "runtime-benchmarks"))]
 		type Origin: From<DidRawOrigin<DidIdentifierOf<Self>>>;
+		#[cfg(feature = "runtime-benchmarks")]
+		type Origin: From<RawOrigin<DidIdentifierOf<Self>>>;
```

Comment:
Mh that's strange. I would like to not have this check here. Maybe we can do some rust magic.

Reply:
Mhh yeah I agree, but the whole benchmarking thing has been somehow "hardcoded" towards using specific functions and keys to "make it work". But hopefully there is a better method.